### DISCUSSION TITLE
Neuer Router

### DISCUSSION THREAD

**Discussion by modulder** [camueller/SmartApplianceEnabler#453] [Installation und Betrieb]
Hallo, 
musste Aufgrund eines Defektes einen neuen Router installieren.

FritzBox 7590Ax ist der Neue.

Ist soweit alles eingerichtet. Ich komme auf den Raspi und auch auf den SHM2.0. Leider findet der SHM keinen neuen Geräte. Kann das mit dem Routertausch zusammen hängen?

Viele Grüße 
Michael

**Comment by camueller** [comment]
Einfach den SHM 2.0 resetten (Reset-Knopf länger als 6s gedrück halten)

**Reply by modulder** [reply]
leider nichts gebracht, ich glaube ich habe noch ein anderes problem, ich würde gerne ihren support in anspruch nehmen.

**Reply by camueller** [reply]
Das klingt eher nach Netzwerk-Problemen, bei denen ich auch nicht helfen kann. Am Raspberry Pi und der SAE-Installation sollte sich ja für den Router-Tausch nichts geändert haben!?

**Reply by modulder** [reply]
https://uploadnow.io/de/share?utm_medium=30759c27-5fa2-4a03-afbf-1dfea88d20db&utm_term=t

ich habe Ihnen mal ein Bild von der LOG gemacht mit "Error" ich hoffe Sie können mir helfen

**Reply by camueller** [reply]
Die Zeilen mit ERROR sollten nicht im Log sein. Da ist irgendetwas am SAE kaputt. Vielleicht sollten wir das doch im Rahmen des Supports klären. Falls Bedarf besteht, bitte eine Email an axel.mueller@avanux.de - da klären wir alles weitere.
